### PR TITLE
[Pods] Update Emission and React Native.

### PR DESCRIPTION
**Pull request by alloy**
As described on the tin.

This branch has been sent off for a beta.

### CODE REVIEW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -57,6 +56,7 @@ upcoming:
       - Changed the paragraph padding on all ARTextView instances - orta
       - Moved ARSwitchView and ARSpinner into new Extraction pod. - alloy
       - Buyer's Premiunm is shown on AuctionInformationViewControllers - orta
+      - Update Emission since first QA. - alloy
```
Comment (orta) [inline CHANGELOG.yml]: might be worth starting to think about a CHANGELOG over there?

File: Podfile [hunk 2]
```diff
@@ -83,6 +83,7 @@ target 'Artsy' do
   # TODO Actually release to our spec repo.
   pod 'Extraction', :git => 'https://github.com/artsy/extraction.git'
   pod 'Emission', :git => 'https://github.com/artsy/emission.git'
+  pod 'React/Core', :git => 'https://github.com/facebook/react-native.git', :tag => 'v0.25.1'
```
Comment (orta) [inline Podfile]: my feedback on "shouldn't emission handle this" from the last one of these still stands ( I disagreed with the comments in the react repo, a `Podfile.lock` is meant to be trusted. It's not like it was 2 years ago.

**Comment by orta** [comment]
👍 on green

File: Podfile (hunk 2 above)
Comment (alloy) [inline Podfile]: That is not the point, though, it's that there is no spec for this version in the master spec repo. The comment on the React repo was about not just using master branch.

I'm still on the fence about copying specs for React to our spec repo.

Comment (orta) [inline Podfile]: OK, cool, I dig.

File: CHANGELOG.yml (hunk 1 above)
Comment (alloy) [inline CHANGELOG.yml]: Agreed

**Pull request merged**